src/modules/Dashboard/features/Applet/Schedule/Schedule.hooks.ts

```diff
@@ -190,15 +193,31 @@ export const usePreparedEvents = (
   } = eventsData ?? {};
 
   useEffect(() => {
-    if (calendarEventsArr) {
-      dispatch(calendarEvents.actions.setCalendarEvents({ events: calendarEventsArr }));
+    if (
+      calendarEventsArr &&
```

Comment:
could yoiu add a variable for the conditions and use early return?

Reply:
done